### PR TITLE
chore(eslint): align lint with prettier rules

### PR DESCRIPTION
**Pull request by frassinier**
**What is the problem this PR is trying to solve?**
ESlint did not match Prettier config regarding `arrow-parens`

**What is the chosen solution to this problem?**
Align ESlint config regarding `arrow-parens` to fit Prettier one

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

Futher read here
* https://eslint.org/docs/rules/arrow-parens
* https://eslint.org/docs/rules/no-mixed-operators

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "extends": "airbnb",
   "parser": "babel-eslint",
   "rules": {
+    "arrow-parens": [2, "as-needed"],
```
Comment (jare-talend) [inline .eslintrc]: Could you modify this file in the tools repository first, please ? https://github.com/Talend/tools/blob/master/tools-javascript/.eslintrc

Comment (frassinier) [reply .eslintrc]: Sure, when it's ok here :)

File: output/forms.eslint.txt [hunk 2]
```diff
@@ -13,5 +13,17 @@ The react/require-extension rule is deprecated. Please use the import/extensions
...
   23:7  error  Form controls using a label to identify them must be programmatically associated with the control using htmlFor  jsx-a11y/label-has-for
 
-✖ 3 problems (3 errors, 0 warnings)
+/home/travis/build/Talend/ui/packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js
+  348:7  error  Expected indentation of 5 tabs but found 6  indent
+  349:6  error  Expected indentation of 4 tabs but found 5  indent
+  807:8  error  Expected indentation of 6 tabs but found 7  indent
+  808:7  error  Expected indentation of 5 tabs but found 6  indent
+
+/home/travis/build/Talend/ui/packages/forms/src/widgets/MultiSelectTagWidget/MultiSelectTagWidget.js
```
Comment (ncomont) [inline output/forms.eslint.txt]: maybe we could remove this rule too from our eslintrc ?

File: .eslintrc (hunk 1 above)
Comment (frassinier) [reply .eslintrc]: It's available [here](https://github.com/Talend/tools/pull/53)

**Pull request merged**